peachjam/models/article.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.text import slugify
+
+
+class Article(models.Model):
+    date = models.DateField(null=False, blank=False)
+    title = models.CharField(max_length=1024, null=False, blank=False)
+    body = models.TextField()
+    author = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    image = models.ImageField(upload_to="articles/")
```

Comment:
As mentioned, we'll need to scope the upload folder based on the pk of the model. See how we do it for BaseAttachment.